iOS-Development/readme.md

```diff
@@ -0,0 +1,103 @@
+## iOS Development
+
+## Introduction
+- [Intorduction to iOS](https://developer.apple.com/tutorials/app-dev-training)
+
+- [Xcode Setup](https://www.youtube.com/watch?v=vKPCvAPW9Ns)
+
+## Introduction To Swift
+- [Swift Book](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html)
+
+- [Swift Course](https://www.youtube.com/watch?v=FcsY1YPBwzQ)
+
+## iOS Development Course
+#### SwiftUI (Recommend First) :
+- [Introduction to SwiftUI](https://developer.apple.com/tutorials/swiftui)
+
+- [100 Days Of SwiftUI](https://www.hackingwithswift.com/100/swiftui)
+
+- [Hacking with SwiftUI](https://www.hackingwithswift.com/books/ios-swiftui)
+
+#### UIKit :
+- [100 Days Of Swift](https://www.hackingwithswift.com/100)
+
+- [Hacking with Swift](https://www.hackingwithswift.com/read)
+
+## View Navigation
+- [SwiftUI Navigation](https://www.kodeco.com/5824937-swiftui-tutorial-navigation)
+
+- [UIKit Navigation](https://www.youtube.com/watch?v=LbAd2FIlnos)
+
+## Networking
+- [Netowking in SwiftUI](https://medium.com/@nutanbhogendrasharma/consume-rest-api-in-swiftui-ios-mobile-app-b3c5d6ecf401)
+
+- [Netowking in UIKit](https://www.freecodecamp.org/news/how-to-make-your-first-api-call-in-swift/)
+
+- [Post API Call](https://www.youtube.com/watch?v=o3Rkg6WmZoY)
+
+- [Advance Networking](https://malcolmkmd.medium.com/writing-network-layer-in-swift-protocol-oriented-approach-4fa40ef1f908)
+
+    #### JSON Coding/Decoding
```

Comment:
Lint error. Please move it to the start of the line

Reply:
I added these spaces on purpose to look like a sub-point.

Reply:
@Hitarthbhatt that is not required. the `#`'s used will automatically make it look like a subpoint.